src/common/pf_lldp.c

```diff
@@ -466,6 +466,28 @@ static void pf_lldp_add_management (
    pf_put_byte (0, PF_FRAME_BUFFER_SIZE, p_buf, p_pos);
 }
 
+/**
+ * Mark lldp peer data as invalid.
+ *
+ * @param net              InOut: The p-net stack instance
+ * @param loc_port_num     In:    Local port number.
+ *                                Valid range: 1 .. PNET_MAX_PORT
+ */
+void pf_lldp_peer_timeout(pnet_t * net, int loc_port_num)
```

Comment:
```suggestion
void pf_lldp_invalidate_peer_info(pnet_t * net, int loc_port_num)
```

Reply:
Done

Reply:
Oups. didi not read comment properly will push another update..

Reply:
Done